### ISSUE TITLE
force vncserver to raw encodong? (faster)

### ISSUE THREAD

**Issue by sujitvasanth** [EXALAB/AnLinux-App#76]
I found you can watch live video over like wifi hotspot, over wifi or on locahost when running fedora  mate with raw encoding over vnc as it does not require cpu usage for encoding. Unfortunately very few vnc clients on android support raw encoding options.

Can you make the vncserver force raw emcoding?

**Comment by exalab999** [comment]
I tried to do this but unfortunately there are no options in tight vnc server to do this.

**Comment by sujitvasanth** [comment]
yep I had a close look at vncserver documentation and it doesn't seem possible.
on android the app Remote Desktop Manager says it uses the default vnc client and gives options for requesting raw encoding from the client end. It works but the initialisiation in to desktop manager is slow. Is there any way of involking the vnc client with some switches either through termux or through intents on android?
you can see te speed difference on ultravnc desktop client which lets you pick the encoding there too.

also is there any way to get sound working? how about rdp or a third party remote desktop solution like nomachine, logmein , teamviewer or splashtop as the server on Linux?
